umbraco-cms-webapp-redis-cache/azuredeploy.json

```diff
@@ -351,150 +392,144 @@
+              "timeAggregation": "Average",
+              "criterionType": "StaticThresholdCriterion"
+            },
+            {
+              "name": "[concat('Microsoft.Web/sites', parameters('appName'), '-403-errors')]",
+              "metricName": "Http403",
+              "dimensions": [],
+              "operator": "GreaterThan",
+              "threshold": 0,
+              "timeAggregation": "Average",
+              "criterionType": "StaticThresholdCriterion"
+            }
+          ],
+          "componentId": "[parameters('appInsightsResource')]",
+          "metricName": "[concat(parameters('appName'), 'server errors')]"
           },
-          "operator": "GreaterThan",
-          "threshold": 0,
-          "windowSize": "PT5M"
-        },
-        "action": {
-          "odata.type": "Microsoft.WindowsAzure.Management.Monitoring.Alerts.Models.RuleEmailAction",
-          "sendToServiceOwners": true,
-          "customEmails": []
-        }
+            "actions": [
+          {
+            "actionGroupId": "[resourceId('microsoft.insights/actionGroups',parameters('actionGroupName'))]"
+          }
+        ]
       }
     },
-    {
-      "apiVersion": "2014-04-01",
-      "name": "[concat('CPUHigh ', variables('appServicePlanName'))]",
-      "type": "microsoft.insights/alertrules",
-      "location": "[variables('appInsightsRegion')]",
+    { 
+      "name": "[concat('The average CPU is high across all the instances of ', variables('appServicePlanName'))]",
+      "type": "Microsoft.Insights/metricAlerts",
+      "apiVersion": "2018-03-01",
+      "location": "global",
       "tags": {
         "[concat('hidden-link:', '/subscriptions/', subscription().subscriptionId,'/resourcegroups/', resourceGroup().name, '/providers/Microsoft.Web/serverfarms/', variables('appServicePlanName'))]": "Resource"
       },
       "dependsOn": [
         "[concat('Microsoft.Web/serverfarms/', variables('appServicePlanName'))]"
       ],
       "properties": {
-        "name": "[concat('CPUHigh ', variables('appServicePlanName'))]",
-        "description": "[concat('The average CPU is high across all the instances of ', variables('appServicePlanName'))]",
-        "isEnabled": false,
-        "condition": {
-          "odata.type": "Microsoft.WindowsAzure.Management.Monitoring.Alerts.Models.ThresholdRuleCondition",
-          "dataSource": {
-            "odata.type": "Microsoft.WindowsAzure.Management.Monitoring.Alerts.Models.RuleMetricDataSource",
-            "resourceUri": "[concat(resourceGroup().id, '/providers/Microsoft.Web/serverfarms/', variables('appServicePlanName'))]",
-            "metricName": "CpuPercentage"
+        "severity": 1,
+        "enabled": true,
+        "scopes": [
+          "[resourceId('Microsoft.Web/serverfarms/', variables('appServicePlanName'))]"
+        ],
+        "targetResourceType" : "Microsoft.Web/serverfarms",
+        "targetResourceRegion": "[resourceGroup().location]",
```

Comment:
this is likely to cause issues if the defaultValue for the location property is not used...  should be parameter('location')